.github/workflows/release.yml

```
@@ -33,13 +38,31 @@ jobs:
         run: npm ci
         timeout-minutes: 2
 
-      - name: Exit pre-release mode (safety check)
-        run: npx changeset pre exit || true
+      - name: Exit pre-release mode and clean up
+        run: |
+          echo "🔄 Ensuring we're not in pre-release mode for main branch..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)